### DISCUSSION TITLE
How to create a user for Cloud Sync

### DISCUSSION THREAD

**Discussion by george-gca** [Future-Scholars/paperlib#490] [Q&A] [answered]
I am following the tutorial about Cloud Sync, but I am stucked at the [Create a User](https://paperlib.app/en/doc/cloud-sync/setup.html#create-a-user) part. I believe the MongoDB site changed since then, and I can't find how to do this step.

**Comment by GeoffreyChen777** [comment]
Hi could you please share me your screenshot of the new webpage?

**Reply by george-gca** [reply]
Currently there is no `Realm Tab`:

![image](https://github.com/Future-Scholars/paperlib/assets/31376482/603d5b77-a8f5-490f-94cb-f15b48d189ce)

**Reply by GeoffreyChen777** [reply]
Now it's `App Services`.

**Reply by george-gca** [reply]
Maybe not

![image](https://github.com/Future-Scholars/paperlib/assets/31376482/8b81812e-5dd3-4ddf-a784-d889639c58e3)

**Reply by GeoffreyChen777** [reply, marked as answer]
Click that card `Paperlib`

**Reply by george-gca** [reply]
Thank you. Now I can continue the setup.
